Assets/UI/CQUICommon.lua

```diff
@@ -7,33 +7,41 @@
 -- ===========================================================================
 --  VARIABLES
 -- ===========================================================================
-local CQUI_ShowDebugPrint = true;
+CQUI_ShowDebugPrint = true;
 
 -- ===========================================================================
 --CQUI setting control support functions
 -- ===========================================================================
 function print_debug(str)
```

Comment:
This is so not flexible - can't pass multiple args. Please use multi args (print_debug(...) and then inside print(...)).